Misc/NEWS.d/next/Core and Builtins/2023-06-09-12-59-18.gh-issue-105549.PYfTNp.rst

```diff
@@ -0,0 +1,2 @@
+Tokenize separately `NUMBER` and `NAME` tokens that are not ambiguous. Patch
```

Comment:
```suggestion
Tokenize separately `NUMBER` and `NAME` tokens and allow 0-prefixed numeric literals that are not ambiguous. Patch
```

Reply:
Looks like this edit might have accidentally been reverted in a force-push? The docs build on Azure Pipelines is now failing